docs/api_example_EN.md

```diff
@@ -0,0 +1,252 @@
+# API application case
```

Comment:
api_example.md for English doc, api_example_cn.md for Chinese doc.

Reply:
done